src/Compilers/CSharp/Portable/Emitter/Model/NamedTypeSymbolAdapter.cs

```diff
@@ -325,7 +325,6 @@ IEnumerable<Cci.MethodImplementation> Cci.ITypeDefinition.GetExplicitImplementat
 
             foreach (var member in this.GetMembers())
             {
-                // PROTOTYPE(refout) Do something here?
```

Comment:
:memo: Explicit interface implementations should be included. My understanding is that they are virtual, and we already concluded that all virtual methods should be included in ref assemblies. I'll add a test to lock that in.